app/views/topology/service.js

```diff
@@ -1346,6 +1349,29 @@ YUI.add('juju-topology-service', function(Y) {
     },
 
     /**
+      Returns a service name which is a maximum of 10 characters wide.
+
+      @method truncateServiceName
+      @param {Object} service The service model
+      @return {String} The truncated service display name.
+    */
+    truncateServiceName: function(service) {
+      var name = service.displayName;
```

Comment:
I also had originally thought about putting it as a method on the model but at this point it's no longer a model. Just the attribute values. I could move it to be an attribute on the model `truncatedDisplayName` which is set via the setter of the `displayName`